tools/test-proxy/Azure.Sdk.Tools.TestProxy/README.md

```diff
@@ -22,13 +22,11 @@ There is a walkthrough through the process below in the [how do I use the test p
 2. Install test-proxy
 
 ```powershell
-> dotnet tool install azure.sdk.tools.testproxy --global --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json
+> dotnet tool install azure.sdk.tools.testproxy --global --prerelease --add-source https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json
```

Comment:
I don't think this works - I was just trying to update to the latest version and there is no supported prerelease option - https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools#install-a-specific-tool-version

Reply:
/cc @mikeharder